### PR TITLE
common/[defg]*: add more information links

### PR DESCRIPTION
**Pull request by marchersimon**
For #5510 
I didn't add a link to `exec`, because I couldn't find any and `ftp`, since it needs a complete rewrite, because it's not about `ftp`.

### CODE REVIEW

**Comment by bl-ue** [comment]
Yeah, `ftp` has a strange history: #1173

File: pages.it/common/dirs.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 > Mostra o manipola uno stack di directory.
 > Uno stack di directory è una lista di directory recentemente visitate che può essere manipolata con i comandi `pushd` e `popd`.
+> Maggiori informazioni: <https://www.gnu.org/software/bash/manual/bash.html\#Directory-Stack-Builtins>.
```
Comment (navarroaxel) [inline pages.it/common/dirs.md:5]: Why this backslash?
```suggestion
> Maggiori informazioni: <https://www.gnu.org/software/bash/manual/bash.html#Directory-Stack-Builtins>.
```

File: pages.ko/common/dirs.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 > 디렉토리 스택을 표시하거나 조작한다.
 > 디렉토리 스택은 `pushd`과 `popd` 명령어로 조작할 수 있는 최근 방문한 디렉토리의 목록이다.
+> 더 많은 정보: <https://www.gnu.org/software/bash/manual/bash.html\#Directory-Stack-Builtins>.
```
Comment (navarroaxel) [inline pages.ko/common/dirs.md:5]:
```suggestion
> 더 많은 정보: <https://www.gnu.org/software/bash/manual/bash.html#Directory-Stack-Builtins>.
```

File: pages.tr/common/dirs.md [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 > Dizin yığını görüntüler veya üzerinde oynama yapar.
 > Dizin yığını, `pushd` ve `popd` komutlarıyla üzerinde oynama yapılabilen, son ziyaret edilen dizinleri gösteren bir listedir.
+> Daha fazla bilgi için: <https://www.gnu.org/software/bash/manual/bash.html\#Directory-Stack-Builtins>.
```
Comment (navarroaxel) [inline pages.tr/common/dirs.md:5]:
```suggestion
> Daha fazla bilgi için: <https://www.gnu.org/software/bash/manual/bash.html#Directory-Stack-Builtins>.
```

File: pages.zh/common/gpg-zip.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # gpg-zip
 
 > 使用`GPG`加密存档中的文件和目录。
-> 更多信息：<https://www.gnupg.org/documentation/manuals/gnupg/gpg_002dzip.html>。
+> 更多信息： <https://www.gnupg.org/documentation/manuals/gnupg/gpg_002dzip.html>.
```
Comment (navarroaxel) [inline pages.zh/common/gpg-zip.md:4]: I think this is the correct period:
```suggestion
> 更多信息： <https://www.gnupg.org/documentation/manuals/gnupg/gpg_002dzip.html>。
```

File: pages/common/dirs.md [hunk 5]
```diff
@@ -2,6 +2,7 @@
 
 > Displays or manipulates the directory stack.
 > The directory stack is a list of recently visited directories that can be manipulated with the `pushd` and `popd` commands.
+> More information: <https://www.gnu.org/software/bash/manual/bash.html\#Directory-Stack-Builtins>.
```
Comment (navarroaxel) [inline pages/common/dirs.md:5]:
```suggestion
> More information: <https://www.gnu.org/software/bash/manual/bash.html#Directory-Stack-Builtins>.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages.zh/common/gpg-zip.md (hunk 4 above)
Comment (bl-ue) [reply pages.zh/common/gpg-zip.md:4]: It is, but #5240 isn't quite done yet, so maybe we should wait on this one. 🤔

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Looks ok to me.

Though I don't see anything wrong with the `ftp` page?

**Comment by sbrl** [comment]
Merging, but we can always open a new PR or issue to discuss any problems this merge might cause.

**Pull request merged**